src/store/getters.js

```diff
@@ -126,11 +110,11 @@ export default {
     },
 
     fan: state => {
-        return Object.entries(state.printer).indexOf('fan') ? state.printer.fan : false;
```

Comment:
let me know if you want me to revert these. They just caught my eye when I started reverting. @meteyou